tools/configure-fwe.sh

```diff
@@ -137,8 +145,14 @@ jq ".staticConfig.mqttConnection.endpointUrl=\"${ENDPOINT_URL}\"" ${INPUT_CONFIG
     | jq ".staticConfig.mqttConnection.decoderManifestTopic=\"${TOPIC_PREFIX}vehicles/${VEHICLE_NAME}/decoder_manifests\"" \
     | jq ".staticConfig.mqttConnection.canDataTopic=\"${TOPIC_PREFIX}vehicles/${VEHICLE_NAME}/signals\"" \
     | jq ".staticConfig.mqttConnection.checkinTopic=\"${TOPIC_PREFIX}vehicles/${VEHICLE_NAME}/checkins\"" \
-    | jq ".staticConfig.mqttConnection.certificateFilename=\"${CERTIFICATE_FILE}\"" \
-    | jq ".staticConfig.mqttConnection.privateKeyFilename=\"${PRIVATE_KEY_FILE}\"" \
+    | if [[ $CERTIFICATE ]]; \
```

Comment:
Causes unbound variable errors when `CERTIFICATE` or `PRIVATE_KEY` not defined, which can be fixed by adding this at the top:
```
if [ -z "${CERTIFICATE+x}" ]; then
    CERTIFICATE=""
fi
if [ -z "${PRIVATE_KEY+x}" ]; then
    PRIVATE_KEY=""
fi
```

Reply:
In bash, `[[ $CERTIFICATE ]]` will check whether variable is set and non zero length. Only in the shell, it will result in unbound variable error. I thought we are using bash at the beginning of the script.

Reply:
I get the following running the script as it is at the moment:
```
~/prj/aws-iot-fleetwise-edge$ ./tools/configure-fwe.sh --input-config-file configuration/static-config.json --output-config-file config.json --vehicle-name abc --endpoint-url https://my-endpoint
./tools/configure-fwe.sh: line 152: PRIVATE_KEY: unbound variable
./tools/configure-fwe.sh: line 148: CERTIFICATE: unbound variable
```

Reply:
Good call. I was testing with the `start-fwe.sh` which had the `--certificate` to the `configure-fwe.sh`.